packages/checkpoint-dev/README.md

```diff
@@ -0,0 +1,6 @@
+# K3d + UDS Core Slim Dev Checkpoint
+
+This is a special modified version of UDS Core that rehydrates K3d + UDS Core Slim Dev from a committed container and volumes.
+
+> [!IMPORTANT]
+> In order to `zarf package create` this package you must have a running UDS k3d cluster with the UDS Core Slim dev installed in it!  This package also requires `sudo` to create and deploy currently - if you see a prompt and it seems stalled it is waiting for password input (hidden by the spinner)
```

Comment:
Can we expand this README with more detail on using this package? It's clear I need a core slim dev installed to create the package, but what about using it? Do I need core slim running to deploy it? I assume so for the "reset", but once it's created can I destroy my k3d cluster and just do `zarf package deploy`? 

Also, I typically deploy core (slim or not) via the `uds` CLI, can I use this package with UDS or do I need to use zarf directly?

Reply:
Added some more info on it - you have to use zarf but it can be `uds zarf` - I added that as the stated command in the README and added some notes to answer those q's